### PR TITLE
feat(console): Enable/Disable API Status dashboard

### PR DESCRIPTION
**Pull request by gcusnieux**
gravitee-io/issues#6365

### CODE REVIEW

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6365-api-status-dashboard/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6365-api-status-dashboard/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

**Review by phiz71 — APPROVED** [review]

File: gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts [hunk 1]
```diff
@@ -17,25 +17,31 @@ import { StateService } from '@uirouter/core';
...
-
-    this.canViewAnalytics = UserService.isUserHasAllPermissions(['environment-platform-r']);
+    if (this.canViewAnalytics) {
+      tabs.push('management.dashboard.analytics');
+    }
+    if (this.canViewAlerts) {
+      tabs.push('management.dashboard.alerts');
+    }
+    const currentTab = tabs.find((tab) => this.$state.is(tab));
+    this.selectedIndex = currentTab ? tabs.indexOf(currentTab) : 0;
```
Comment (ytvnr) [inline gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts]:
```suggestion
     this.selectedIndex  = tabs.findIndex((tab) => this.$state.is(tab));
```

File: gravitee-apim-console-webui/src/management/configuration/portal/portal.html [hunk 2]
```diff
@@ -107,6 +107,20 @@ <h3>API labels dictionary</h3>
...
       </md-input-container>
 
+      <h3>Dashboard</h3>
+      <md-input-container class="gv-input-container-dense">
+        <md-checkbox
+          ng-model="$ctrl.settings.dashboards.apiStatus.enabled"
+          aria-label="Enable API status panel"
+          ng-disabled="$ctrl.isReadonlySetting('console.dashboards.apiStatus.enabled')"
+        >
+          Enable API STATUS panel
```
Comment (ytvnr) [inline gravitee-apim-console-webui/src/management/configuration/portal/portal.html]:
```suggestion
          Display API STATUS panel
```

**Review by ytvnr — APPROVED** [review]
two suggestions but good to me

File: gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts (hunk 1 above)
Comment (gcusnieux) [reply gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts]: Not exactly the same, but okay 😄

Comment (gcusnieux) [reply gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts]: We don't want selectedIndex to be equals -1

**Comment by graviteeio** [comment]
**Gravitee.io Automatic Deployment**

Congrats, all the CI steps were 🟢 . The Bot was able to deploy the build of this PR, so you can now try it directly [here](https://apimnightlywebui24386.blob.core.windows.net/6365-api-status-dashboard/index.html)! 🚀

_Notes_: The deployed app is linked to the management API of the Element Zero team's environment.

File: gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts (hunk 1 above)
Comment (ytvnr) [reply gravitee-apim-console-webui/src/management/dashboard/dashboard.controller.ts]: ahah good catch

**Pull request merged**